### PR TITLE
cpu/native: adjusted stacksizes

### PR DESCRIPTION
**Pull request by haukepetersen**
The stacksizes for native were defined in a way, so that DEBUG did not work inside of interrupt routines. The new values should still be large enough but allow now for this.

### CODE REVIEW

**Comment by LudwigOrtmann** [comment]
@josephnoir You have an OS X computer, right? Can you test please?

**Comment by josephnoir** [comment]
Yes, I do. How can I test this, just run all thread tests? (Sorry, I sometimes miss the notifications)

**Comment by josephnoir** [comment]
Well, to no surprise, I run into the same problem as the tester on #2870

**Comment by haukepetersen** [comment]
but did you have trouble just for the vtimer test or also for other test cases?

**Comment by josephnoir** [comment]
Considering the error message:
```
/some/long/path/usr/include/mach/task.h:193:15: note: 
      previous declaration is here
kern_return_t thread_create
              ^
1 error generated.
```
I ran the `thread_basic` test and added that observation to #3175.

**Comment by LudwigOrtmann** [comment]
@haukepetersen how about just not changing the OSX stack sizes by an order?

**Comment by LudwigOrtmann** [comment]
(... maybe add a TODO comment in there instead)

**Comment by haukepetersen** [comment]
Ok, To be really on the save side, I just lowered the printf_extra values slightly (by 64byte). This should be very save now. And the TODO that is already in the comments still holds.

**Comment by kaspar030** [comment]
Could you summarize shortly the reason for this change?

**Comment by haukepetersen** [comment]
> The stacksizes for native were defined in a way, so that DEBUG did not work inside of interrupt routines.

**Comment by kaspar030** [comment]
> The stacksizes for native were defined in a way, so that DEBUG did not work inside of interrupt routines.

Answers. I want answers.

**Comment by haukepetersen** [comment]
I thought it was. This check
```
sched_active_thread->stack_size > THREAD_EXTRA_STACKSIZE_PRINTF
```
in `debug.h` leads to interrupts not being able to output debug information in native, since the interrupt context is running with `THREAD_STACKSIZE_DEFAULT` as far as I saw. Since in the existing code, `SACKSIZE_DEFAULT` is equal to `EXTRA_STACKSIZE_PRINTF`, you do not get debug output from interrupt routines. Now making the printf stacksize smaller than the default stacksize fixes this.

**Comment by LudwigOrtmann** [comment]
@haukepetersen please rebase on master

**Comment by haukepetersen** [comment]
rebased.

**Comment by authmillenon** [comment]
@LudwigOrtmann was your label-setting an implicit ACK(

**Comment by authmillenon** [comment]
(still needs some rebase though as travis has some old repository information)

**Comment by OlegHahm** [comment]
@LudwigOrtmann, ping?

**Comment by OlegHahm** [comment]
@LudwigOrtmann cannot remember anymore.

**Comment by kaspar030** [comment]
Tested on Linux and OS X, and strider is happy. As it's a minimal constant change, will skip travis. Go.
